.github/workflows/e2e.frontend.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
           - groups/groups.spec.ts
           - projects/access.spec.ts
           - segments/segments.spec.ts
-          - login/login.spec.ts
+          #- login/login.spec.ts
```

Comment:
@Tymek this spec seems to be new and it's not working in CI. I'm commenting it out for now.